src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -99,15 +99,15 @@ export class BaseEditCourseForm extends React.Component {
       return [];
     }
 
-    return data.actions.PUT;
+    return data.actions.POST;
```

Comment:
?

Reply:
This is because OPTIONS returns the set of actions for the default expected HTTP verb. On a course-detail endpoint, that's PUT (update course), but on a course-list endpoint, that's POST (new course).